sys/src/libacpi/acfileio.c

```diff
@@ -125,6 +125,7 @@
 
 /* Local prototypes */
 
+#if 0
```

Comment:
This file consists mostly of functions with their contents #if 0'd.  Are you suggesting getting rid of the function and the prototypes?